requirements/base.in

```diff
@@ -8,3 +8,4 @@ djangorestframework
 edx-django-utils
 edx-opaque-keys
 openedx-atlas
+openedx-completion-aggregator
```

Comment:
Nit: please add a comment explaining why we need this dependency.

Reply:
Added.